packages/node-core/src/indexer/store.service.ts

```diff
@@ -117,6 +117,7 @@ export class StoreService {
       logger.warn(`Historical feature is not support with ${this.dbType}`);
     }
     this.storeCache.setHistorical(this.historical);
+    this.storeCache.setUseCockroach(this.dbType === SUPPORT_DB.cockRoach);
```

Comment:
Maybe these could be combined into an init?

Reply:
> This just fixes the deferrable issue right? But we still have the issue with constraints failing because of the order of queries?

This fixes the deferrable issue. constraint with query order issue could still be an issue, test with some relation entity project under substrate appears working, will need to check again with the other network projects to confirm.